Mesh_3/doc/Mesh_3/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -246,6 +252,11 @@ From the example (\ref Mesh_3/mesh_3D_image.cpp):
 
 \snippet Mesh_3/mesh_3D_image.cpp Domain creation
 
+From the example (\ref Mesh_3/mesh_3D_weighted_image.cpp),
+where the labeled image is used with precomputed weights image :
```

Comment:
For a consistent wording, assuming "image of weights" is the right wording:
```suggestion
From the example (\ref Mesh_3/mesh_3D_weighted_image.cpp),
where the labeled image is used with a precomputed image of weights:
```